runner/entrypoint.sh

```diff
@@ -3,6 +3,12 @@
 RUNNER_ASSETS_DIR=${RUNNER_ASSETS_DIR:-/runnertmp}
 RUNNER_HOME=${RUNNER_HOME:-/runner}
 
+if [ "${RUNNER_CUSTOM_RBAC:-}" == "true" ]; then
+  # Let runner execute these hooks
+  export ACTIONS_RUNNER_HOOK_JOB_STARTED=/job_start.sh
+  export ACTIONS_RUNNER_HOOK_JOB_COMPLETED=/job_end.sh
+fi
```

Comment:
```suggestion
# Let runner execute these hooks
export ACTIONS_RUNNER_HOOK_JOB_STARTED=/job_start.sh
export ACTIONS_RUNNER_HOOK_JOB_COMPLETED=/job_end.sh
```

I think it's fine to enable these hooks by default because we skip the status update anyway when the envvar is not specified.

Reply:
AFAIK we can only specify only one script per hook, and that makes me wonder how existing users already using the hooks to leverage this new runner status update hooks.

Perhaps we'd better introduce another layer of indirection, so that existing users can put their own hooks in a predefined location, and ARC's default hooks finds and runs those custom hooks along with these `job_start.sh` and `job_end.sh`?

For example, how about making `job_start.sh` look like:

```
for hook in $(ls /hooks/job_started); do
  $hook "$1" "$2"
done
```

And we put e.g. `job_start.sh` into `/hooks/job_started/job_start.sh`?

Reply:
I'm merging this together with @Fleshgrinder 's idea from https://github.com/actions-runner-controller/actions-runner-controller/pull/1268#discussion_r874052393